src/Raven.Server/Smuggler/Documents/SmugglerPatcher.cs

```diff
@@ -45,6 +45,9 @@ public Document Transform(Document document)
                     }
                 }
 
+                if (translatedResult is Document d)
+                    return d.Clone(ctx);
```

Comment:
Why we didn't need to have it before? Since this is in the smuggler then it's not related to projection query, right?

Reply:
I changed the `ScriptRunner` so it may return a `Document` instance, instead of always creating a new blittable. So we had to that that into account here as well

Reply:
Can't we call `Clone()` inside `SriptRunner` then?

Reply:
We usually don't need to.
See below here, where we clone the object we get if it isn't a `Document` as well

Reply:
OK